examples/simple/main.tf

```
@@ -0,0 +1,54 @@
+#
+# WAF
+#
+
+resource "aws_wafregional_rate_based_rule" "ipratelimit" {
```

```suggestion
resource "aws_waf_regional_rate_based_rule" "ipratelimit" {
```

It's a nitpick but I don't understand why there isn't an underscore there

---

That's how the terraform AWS provider names it, I can't change this.